### ISSUE TITLE
Edited Staff from Monthly to Hourly, got reflected only after closing the app and opening again. The change should be reflected while the admin is in the app

### ISSUE THREAD

**Issue by HazelColaco19** [ajency/Dhanda-App#111]
https://drive.google.com/file/d/1JK0l0bA9RnAIcd2_2-M6_g3o6RTqZrGq/view?usp=drivesdk

**Comment by ShubhamModi004** [comment]
Edit staff button will stop working after staff page implementation.

**Comment by HazelColaco19** [comment]
This does not apply anymore since we have kept payroll type change on Hold.
